bin/luminork/src/main.rs

```diff
@@ -101,7 +91,13 @@ async fn async_main() -> Result<()> {
         )
         .await
     } else {
-        let config = Config::try_from(args)?;
+        debug!("creating innit-client...");
+        let provider = Some((
+            InnitClient::new(InnitConfig::builder().build()?).await?,
+            get_host_environment(),
+        ));
```

Comment:
I don't super love doing this here and it may move as we'll need to pass some actual config to innit or have it discover it sing env vars